libs/application/templates/parental-leave/src/forms/ParentalLeaveForm.ts

```diff
@@ -388,21 +388,12 @@ export const ParentalLeaveForm: Form = buildForm({
           id: 'employer',
           title: parentalLeaveFormMessages.employer.subSection,
           children: [
-            buildRadioField({
+            buildCustomField({
+              component: 'SelfEmployed',
               id: 'employer.isSelfEmployed',
               title: parentalLeaveFormMessages.selfEmployed.title,
               description: parentalLeaveFormMessages.selfEmployed.description,
               width: 'half',
```

Comment:
Dont think you need to specify `width: 'half'` here as well as you have it in the custom component.